hw/ip/kmac/data/kmac.hjson

```diff
@@ -308,6 +315,61 @@
         }
       ]
     } // R: STATUS
+    { name: "ENTROPY_PERIOD"
+      desc: '''Entropy Refresh Period.
+
+      This register determines the refresh period. The value is the number of
+      clock cycles. In every entropy period, the entropy generation logic in
+      KMAC requests new entropy to EDN and feed the value into LFSR seed.
+
+      If 0, the entropy generation logic does not refresh its seed. The
+      software manually updates the seed by writing into "ENTROPY_SEED"
+      registers.
+      '''
+      swaccess: "rw"
+      hwaccess: "hro"
+      regwen: "CFG_REGWEN"
+      fields: [
+        { bits: "31:0"
+          name: "period"
+          desc: "entropy period in clock cycles"
+        }
+      ]
+    } // R: ENTROPY_PERIOD
+    { name: "ENTROPY_SEED_LOWER"
+      desc: '''Entropy Seed [31:0].
+
+      Everytime when the software writes into this register, LFSR is reseeded
+      with the value of {ENTROPY_SEED_UPPER, ENTROPY_SEED_LOWER}.
+      '''
+      swaccess: "rw"
+      hwaccess: "hro"
+      hwqe: "true"
+      regwen: "CFG_REGWEN"
```

Comment:
does it make sense for this one to be locked by `cfg_regwen`? because i can imagine the software wanting to lock all other configurations and just repeatedly updating the seed.

Reply:
I didn't expect the software keep changing the seed in a KMAC block. Actually, I also think that just preparing one 1600 bit and re-use same entropy for all 24 rounds is also fine...